website/src/data/gallery.json

```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "title": "AutoTx - Crypto Transactions Agent",
+    "link": "https://blog.polywrap.io/p/autotx-your-ai-powered-transaction",
+    "description": "Generates on-chain transactions, which are submitted to a smart account so users can easily approve & execute them.",
+    "image": "autotx.png",
+    "tags": ["tools", "blockchain"]
```

Comment:
```suggestion
    "tags": ["tools", "groupchat", "app", "blockchain"]
```

Reply:
@skzhang1 @victordibia Is there an issue with using more than two tags?
There is "We recommend using no more than two tags for clarity" in https://microsoft.github.io/autogen/docs/Gallery/#contributing

Reply:
It will not trigger technique issues. We make this illustration just for clarity purpose. These four tags are distinct and I think it is good.